argus/backend/service/jenkins_service.py

```diff
@@ -51,7 +51,16 @@ def __init__(self) -> None:
                                         password=current_app.config["JENKINS_API_TOKEN"])
 
     def retrieve_job_parameters(self, build_id: str, build_number: int) -> list[Parameter]:
-        job_info = self._jenkins.get_build_info(name=build_id, number=build_number)
+        if build_number == -1:
```

Comment:
I believe -1 is just not known/non existent - better use `build_number: Optional[int]` and use `None`

Reply:
Done